x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_was_authorized_details.test.tsx

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { mountWithIntl, shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { BrowserFields } from 'x-pack/plugins/secops/public/containers/source';
+
+import { mockBrowserFields } from '../../../../containers/source/mock';
+import { mockEcsData, TestProviders } from '../../../../mock';
+
+import {
+  AuditdWasAuthorizedDetails,
+  AuditdWasAuthorizedLine,
+} from './auditd_was_authorized_details';
+
+describe('AuditAcquiredCredsDetails', () => {
+  describe('rendering', () => {
+    test('it renders the default AuditAcquiredCredsDetails', () => {
+      // I cannot and do not want to use the mocks for the snapshot tests as they are too heavy
```

Comment:
> // `I cannot and do not want to use the mocks for the snapshot tests as they are too heavy`

4 lines below this comment, [on line 27](https://github.com/elastic/kibana/pull/33470/files#diff-cebeaf933113be9a365aa35c6d21b69fR27) the test references `mockEcsData[21]`:

```
<AuditdWasAuthorizedDetails browserFields={browserFields} data={mockEcsData[21]} />
```

imported via

```
import { mockEcsData, TestProviders } from '../../../../mock';
```

and then asserts [on line 30](https://github.com/elastic/kibana/pull/33470/files#diff-cebeaf933113be9a365aa35c6d21b69fR30):

```
expect(toJson(wrapper)).toMatchSnapshot();
```

so at a glance, it looks like the snapshot test is using mocks. Perhaps, I'm missing something subtle about the mock data itself?

Reply:
As discussed, yes, the missing part is that the documentation is just not clear for the comment. It is not using mocks for the `const browserFields: BrowserFields = {};`